app/main.py

```diff
@@ -1,13 +1,21 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors": [format_linter_error(error) for error in errors],
+        "path": file_path,
+        "status": "failed" if errors else "passed"
+    }
 
 
-def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+def format_linter_report(lin_rep: dict) -> list:
+    return \
+        [format_single_linter_file(file, lin_rep[file]) for file in lin_rep]
```

Comment:
try not to use slash here, it is better to make this way, but with list
![image](https://user-images.githubusercontent.com/103201010/215846127-ed7ee557-0dca-4a08-9415-b17f5199d61e.png)

Reply:
Fixed (without slash) .